docs/components.stories.js

```diff
@@ -1,4 +1,39 @@
-export const search_demo = `
+import { storiesOf } from '@open-wc/demoing-storybook';
+import '../src/index';
+
+
+// display the html code above the active display
+const wrapInHtmlCode = (html_example) => { 
```

Comment:
this is currently showing a primitive way to show the html directly before the real thing: 
![example of wrapping code in its example before showing](https://user-images.githubusercontent.com/5565284/70939805-0517bd80-2017-11ea-95b2-23534beaa916.png)

Reply:
outmoded and improved upon